tests/dummy/app/adapters/application.js

```diff
@@ -3,4 +3,4 @@
 import AdapterFetch from 'ember-fetch/mixins/adapter-fetch';
 import JSONAPIAdapter from '@ember-data/adapter/json-api';
 
-export default JSONAPIAdapter.extend(AdapterFetch);
+export default class extends JSONAPIAdapter.extend(AdapterFetch) {}
```

Comment:
This will conflict with
https://github.com/adopted-ember-addons/ember-data-factory-guy/pull/464

But whoever merges last can handle the conflct